bin/p2-rctl/main.go

```diff
@@ -117,17 +117,11 @@ func main() {
 
 	rcStore := rcstore.NewConsul(client, labeler, 3)
 	rctl := rctlParams{
-		httpClient: httpClient,
-		baseClient: client,
-		// rcStore copied so many times right now but might not all be
```

Comment:
you could probably keep these three lines if you want, an donly delete the `rcWatcher` parts